pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -719,3 +519,23 @@ func (exp *flowExporter) findFlowType(conn flowexporter.Connection) uint8 {
 	klog.Warningf("Source IP: %s doesn't exist in PodCIDRs", conn.FlowKey.SourceAddress.String())
 	return 0
 }
+
+func (exp *flowExporter) exportConn(conn flowexporter.Connection) error {
+	// TODO: more records per data set will be supported when go-ipfix supports size check when adding records
+	if err := exp.addConnToSet(conn); err != nil {
+		return err
+	}
+	if _, err := exp.sendDataSet(); err != nil {
+		return err
+	}
+	exp.numDataSetsSent = exp.numDataSetsSent + 1
+	klog.V(4).InfoS("Record for connection sent successfully", "flowKey", conn.FlowKey, "connection", conn)
+	return nil
+}
+
+func (exp *flowExporter) getMinTime(t1, t2 time.Duration) time.Duration {
```

Comment:
does it have to be the method of the struct?

Reply:
Have moved the method to utils.go, if that's fine? The purpose for having a separate method out is to make `sendFlowRecords` clean.

Reply:
As `exporter.go` is the only where this function is used, we can keep it here itself but as a standalone function.